csm_web/frontend/src/components/section/SpacetimeEditModal.tsx

```diff
@@ -36,6 +38,7 @@ export default class SpacetimeEditModal extends React.Component<SpacetimeEditMod
       time: timeString.slice(0, sliceIndex),
       isPermanent: false,
       changeDate: "",
+      override: props.overrideEdit,
```

Comment:
I don't believe there is any need to make this a state; this variable won't be changing in the lifecycle of the component, and is always going to be the same as the prop.